### ISSUE TITLE
Android - Home Screen - TalkBack Reads Hidden "About" Pop-up Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#19]
When using TalkBack to read the "Steps" on the home screen, it reads the “Step 8” text and then reads the three items that show up when you activate the “About” button. Once TalkBack reads the three items it then reads the four items in the footer. I would only expect TalkBack to read that text when the "About" button/link is activated and not while it is closed.

**Comment by KyleLivingstonDOL** [comment]
Hi John,

We've updated this and will be demo'ed on the next 508 compliance check after Sprint 7.

Best,
Kyle

**Comment by jfc3-dol** [comment]
When does sprint 7 end and when I will see the application, give or take a day or two?

**Comment by KyleLivingstonDOL** [comment]
> When does sprint 7 end and when I will see the application, give or take a day or two?

UAT for Sprint 7 (which is ongoing right now) will start on June 10.

**Comment by jfc3-dol** [comment]
Closing issues because TalkBack now does not read the three hidden items that pop-up when you activate the "About" button.
